flowredux/src/commonMain/kotlin/com/freeletics/flowredux/sideeffects/Action.kt

```diff
@@ -6,9 +6,16 @@ import com.freeletics.flowredux.dsl.reduce
 internal sealed class Action<S, A>
```

Comment:
nitpicking: but we could make this non null:`S:Any, A:Any`

Reply:
I'll get rid of this completely, so I kept it like this for now. It would trigger a ton of changes for SideEffect and the builders as well